packages/story-editor/src/components/panels/design/sizePosition/radius.js

```diff
@@ -77,45 +73,32 @@ const inputContainerStyleOverride = css`
   }
 `;
 
-const styleOverrideTopLeft = css`
-  ${inputContainerStyleOverride}
-  border-radius: 4px 0 0 4px;
+const iconCss = css`
+  width: 29px;
+  height: 29px;
 `;
 
-const styleOverrideTopRight = css`
-  ${inputContainerStyleOverride}
-  border-radius: 0;
+const StackableInput = styled(NumericInput)`
+  svg {
+    ${iconCss}
+  }
+  & > div {
+    border-radius: 0;
+    padding: 2px 7px 2px 12px;
+    ${inputContainerStyleOverride};
```

Comment:
You need this one in the border panel as well (or actually in the generalized StackableInput as suggested elsewhere) as it includes the `:focus-within { z-index: 1; }` which sends the focused input to the top avoiding the slight clipping of the focus outline.

Compare the focus outlines here, where you see the outline being behind the neighbor for the border, but not for the radius:

| Radius panel | Border panel |
|-|-|
| ![Screen Shot 2021-11-03 at 14 04 04](https://user-images.githubusercontent.com/637548/140186472-30395323-6154-4c86-a752-befffb05cae1.png) | ![Screen Shot 2021-11-03 at 14 03 52](https://user-images.githubusercontent.com/637548/140186483-55fa0f78-387a-46ef-8504-d28ac1f77190.png) |

(notice the two little notches in the right side of the focus outline in the second screenshot)